978-F.68/T-SP-F.68-2011-MSW-E.adoc

```diff
@@ -0,0 +1,707 @@
+= List of Telex Destination Codes (TDC) and Telex Network Identification Codes (TNIC)
+:bureau: T
+:docnumber: 978-15.IV.2011
+:title: LIST OF TELEX DESTINATION CODES (TDC) AND TELEX NETWORK IDENTIFICATION CODES (TNIC)
+:series: 
+:series1: COMPLEMENT TO ITU-T RECOMMENDATIONS
+:series2: F.69 (06/1994) AND F.68 (11/1988)
```

Comment:
Instead of:
```
:series:
:series1: ...
:series2: ...
```

It should probably be a new syntax, such as:
```
:complements: F.69 (06/1994), F.68 (11/1988)
```

Reply:
Changes applied